### DISCUSSION TITLE
Use icon by css class..?

### DISCUSSION THREAD

**Discussion by madcoda9000** [microsoft/fluentui-blazor#761] [Q&A] [answered]
Hello,

is it possible to use an icon from the fluent icons by an css class in html?

For ex.:

`<i class="fluentIcon clock" style="color:var(--accent-base-color);"></i>`

Best Regards
Sascha

**Comment by vnbaaij** [comment, marked as answer]
No, this is not possible. 

There are about 13k icons in the library so it would become a pretty hefty CSS file

**Reply by madcoda9000** [reply]
ok, thank you for the answer :-(

**Comment by dvoituron** [comment]
Hi,

For complete information, you can use this **FluentUI System Icons Font** to include the font and associated CSS.
https://github.com/microsoft/fluentui-system-icons/tree/main/fonts

But this is not a "Blazor" solution :-)

**Reply by vnbaaij** [reply]
Good addition!

But it also proves what I said about the CSS. Almost 22k lines just for the regular icons...

**Reply by madcoda9000** [reply]
> Hi,
> 
> 
> 
> For complete information, you can use this **FluentUI System Icons Font** to include the font and associated CSS.
> 
> https://github.com/microsoft/fluentui-system-icons/tree/main/fonts
> 
> 
> 
> But this is not a "Blazor" solution :-)

Thank you for that additional information!

Best regards 
Sascha
